Backpack/src/main/java/net/skyscanner/backpack/calendar/view/MonthView.kt

```diff
@@ -204,6 +209,26 @@ internal class MonthView @JvmOverloads constructor(
     numberOfRows = calculateNumRows()
   }
 
+  @VisibleForTesting
+  fun getNonDrawnDaysOffset(): Int {
```

Comment:
That will be made `private` at compile time, as by default side-effect of the `@VisibleForTesting` annotation.

Reply:
I don't think it does anything by default, it's just a way to tell developers it shouldn't be used. I've tested adding it to the 'text' field in `BpkDialog` and it is still possible to change it in the example app.

Plus I personally prefer to use `internal` because is a language feature and there is less confusion about what it does.

Reply:
Internal added.